etc/skydive.yml.default

```diff
@@ -262,6 +262,11 @@ storage:
     # The number of indices to keep before deleting.
     # A value of 0 specifies no limit (i.e. indices will never be deleted)
     # indices_to_keep: 0
+    
+    # NOTE: you can override with specific values any of the above values
```

Comment:
I'm really not a big fan of overriding flow, topology, etc here. What about named backend/connection that you can pick to use in the backend parameter, like

```
storage:
   es_1:
      type: elasticsearch
      maxconns: 200
   es_2:
      type: elasticsearch
      maxconns: 400

analyzer:
   flow:
      backend: es_2
   topology:
      backend: es_1
```

Reply:
This raise two separate issues:

(1) should we use defaults? (such as storage.elasticsearch.flow and fallbacking to storage.elasticsearch) - as this significantly reduces the size of configuration I think this makes a lot of sense.

(2) to we use labels for backend, so as backend can also be memory or orientdb you would rather have to support values such as:

```
analyzer:
  flow:
    # backend: memory
    # backend: orentdb
    # backend: elasticsearch
    # backend: elasticsearch:es_2
```
But I am not sure this extra flexibility is required ... (as compared with defining backend:elasticsearch and having the logic of where to look for configuration coded within the code)